<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Allen Winter &lt;winter@kde.org&gt;</author>
 <comment>Configuration page for the Korganizer Event Summary</comment>
 <class>ApptSummaryConfig_Base</class>
 <widget class="QWidget" name="ApptSummaryConfig_Base">
  <property name="windowTitle">
   <string>Event Summary Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QGroupBox" name="mDaysButtonGroup">
     <property name="title">
      <string>Show Upcoming Events Starting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="mDateTodayButton">
        <property name="toolTip">
         <string>Show events for today only</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to see events occurring on this date only.</string>
        </property>
        <property name="text">
         <string>&amp;Today only</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mDateMonthButton">
        <property name="toolTip">
         <string>Show events starting within the next month</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want to see events that start sometime during the next 31 days.</string>
        </property>
        <property name="text">
         <string>Within the next &amp;month (31 days)</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="mDateRangeButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Select the days for showing upcoming events</string>
          </property>
          <property name="whatsThis">
           <string>Check this box if you want to specify the number of days in the future for upcoming events.</string>
          </property>
          <property name="text">
           <string>Within the &amp;next:</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="mCustomDays">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Set the number of days to show upcoming events</string>
          </property>
          <property name="whatsThis">
           <string>Use this spinbox to set the number of days to show upcoming events up to 1 year in the future.</string>
          </property>
          <property name="specialValueText">
           <string>1 day</string>
          </property>
          <property name="suffix">
           <string comment="days to show in summary"> days</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>365</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>7</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>1</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="mShowButtonGroup">
     <property name="title">
      <string>Show These Upcoming Events From Your Calendar</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="mShowBirthdaysFromCal">
        <property name="toolTip">
         <string>Show birthdays from your calendar when also using the Birthday resource</string>
        </property>
        <property name="whatsThis">
         <string>Enable this option to include birthdays from your calendar in the upcoming events summary. This option is only available if you are using the Birthdays resource in your calendar.</string>
        </property>
        <property name="text">
         <string>Show &amp;birthdays</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mShowAnniversariesFromCal">
        <property name="toolTip">
         <string>Shows anniversaries from your calendar when using the Birthdays resource</string>
        </property>
        <property name="whatsThis">
         <string>Enable this option to include anniversaries from your calendar in the upcoming events summary. This option is only available if you are using the Birthdays resource in your calendar.</string>
        </property>
        <property name="text">
         <string>Show &amp;anniversaries</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Groupware Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="mShowMineOnly">
        <property name="toolTip">
         <string>Show events belonging to my calendars only</string>
        </property>
        <property name="whatsThis">
         <string>Check this box if you want the summary to show events belonging to your calendars only.</string>
        </property>
        <property name="text">
         <string>Show &amp;my events only</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
